### PR TITLE
Prakash

### PR DESCRIPTION
**Pull request by v-prkali**

### CODE REVIEW

File: CRM Solution Manager/CRM Solution/App.config [hunk 1]
```diff
@@ -4,23 +4,6 @@
     <add key="OrgServiceUrl" value="https://test.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
     <add key="DynamicsUserName" value="test@test.onmicrosoft.com" />
     <add key="DynamicsPassword" value="*****" />
-    <add key="GitUserName" value="v-raras" />
-    <add key="GitPassword" value="*****" />
```
Comment (amrullah08) [inline CRM Solution Manager/CRM Solution/App.config]: let git details be there in app.config not in configuration

File: CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs [hunk 2]
```diff
@@ -137,7 +156,56 @@ public static string BranchName
...
+        /// <param name="retrievedConfigurationSettingsList">entity collection</param>
+        public override void SetRepositoryConfigurationProperties(EntityCollection retrievedConfigurationSettingsList)
+        {
+            foreach (Entity setting in retrievedConfigurationSettingsList.Entities)
+            {
+                string key = setting.GetAttributeValue<string>("syed_name");
+
+                switch (key)
+                {
+                    case "RepositorySolutionFolder":
```
Comment (amrullah08) [inline CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs]: move all these keys "RepositorySolutionFolder" and other keys from string hardcodings to a constants files

Comment (amrullah08) [reply CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs]: do this changes everywhere

File: CRM Solution Manager/CRM Solution/App.config (hunk 1 above)
Comment (v-prkali) [reply CRM Solution Manager/CRM Solution/App.config]: done.

File: CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs (hunk 2 above)
Comment (v-prkali) [reply CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs]: done.

**Pull request merged**